### ISSUE TITLE
image description under the photo

### ISSUE THREAD

**Issue by Michalf94** [vvvmax/unitegallery#90]
Hi:> How can I change position of the text-panel with description? I would like to have description under the picture.

Thanks!

**Comment by vvvmax** [comment]
HI, this option exists, but it's not documented yet. Sorry. I'll try to
make documentation tomorrow / after tomorrow.

Basically the option is: tile_textpanel_position:"bottom",

**Comment by Michalf94** [comment]
Thanks for reply
I check it:
`  jQuery("#gallery").unitegallery({
tile_textpanel_position:"bottom",`

but it doesn`t work. The text panel is on the top of lightbox (ex: https://s31.postimg.org/l4s1flgsb/Przechwytywanie.jpg)

Is it another way to change it?

**Comment by vvvmax** [comment]
Oh. Thoght you ate talking about tile text panel. The wide lightbox has
only top panel. For bottom panel please use compact lightbox
lightbox_type:"compact"

**Comment by Michalf94** [comment]
Thanks very much for your really fast reply!
Yes! It is work now. Is it possible to have tittle under the photo and description on bottom? 
Maybe should I change sth in CSS?

**Comment by vvvmax** [comment]
I dont have such option, sorry. With css would be difficult.

**Comment by Michalf94** [comment]
Ok, thanks. What about text format of the description? Is it possible to change color or font in CSS?

**Comment by vvvmax** [comment]
Yes i have options documented and what dont have options you can do with
simple css. You can use firebug to trace the lightbox yext class
